src/resources/assets/js/MagicSamMixin.vue

```diff
@@ -39,6 +39,12 @@ export default {
         },
     },
     methods: {
+        set_throttleInterval(interval){
+            this.throttleInterval = interval;
+            if (magicSamInteraction) {
+                magicSamInteraction.throttleInterval=interval;
```

Comment:
Please implement a setter for this to be consistent with similar OpenLayers interactions: `magicSamInteraction.setThrottleInterval(interval)`.